src/main/webapp/index.html

```diff
@@ -16,7 +16,18 @@
   </head>
   <body>
     <div id="content">
-
+      <!--TODO: Add a navbar-->
+      <p id="welcome">Welcome to InterviewMe!</p>
+      <p>InterviewMe is a platform for conducting mock technical interviews where interviewers and interviewees alike can 
+        practice their skills. We aim to connect users with live video chats in order to encapsulate the true feeling
+        and pace of a technical interview.
+      </p>
+      <br>
+      <p>If you would like to logout, please click here.</p>
```

Comment:
l think we should incorporate login/logout as one portion of the navbar. Delete this?

Reply:
Done